app/code/Magento/Developer/Test/Unit/Console/Command/GeneratePatchCommandTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Developer\Test\Unit\Console\Command;
+
+use Magento\Developer\Console\Command\GeneratePatchCommand;
+use Magento\Framework\Component\ComponentRegistrar;
+use Magento\Framework\Filesystem\Directory\ReadFactory;
+use Magento\Framework\Filesystem\Directory\WriteFactory;
+use Magento\Framework\Filesystem\DirectoryList;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class GeneratePatchCommandTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @var ComponentRegistrar
+     */
+    private $componentRegistrar;
+
+    /**
+     * @var DirectoryList
```

Comment:
Please, import `PHPUnit\Framework\MockObject\MockObject;` and use it for documenting properties
```suggestion
     * @var DirectoryList|MockObject
```